rsts/concepts/data_management.rst

```diff
@@ -4,9 +4,28 @@
 Understand How Flyte Handles Data
 #################################
 
+Let's understand how Flyte handles data management.
+
 Types of Data
 =============
 
+There are two parts to the data in Flyte:
+
+1. Metadata
+
+* It consists of data about inputs to a task, and other artifacts.
+* It is configured globally for FlytePropeller, Admin etc, and the running pods/jobs need access to this bucket to get the data.
+
+2. Raw data
+
+* It is the actual data (such as the pandas dataframe, spark dataframe, etc,).
+* Raw data paths are unique for every execution, and the prefixes can be modified per execution.
+* None of the Flyte control plane components would access the raw data. This provides a great separation of data between the control plane and the data plane.
+
+.. note:
+  Metadata and raw data can be present in entirely separate buckets.
+
+
```

Comment:
@SmritiSatyanV, could you let me know where you learned this from? Just wanted to make sure it's correct and not repeated.